clarify #499 (#514)
solidjs/solid-docs-next: src/routes/concepts/understanding-jsx.mdx

#### Core concepts

> - **Static props**:
>   In Solid's JSX, static props are integrated directly into the HTML template as attributes. The template is cloned and used as is.
> 
> - **Dynamic props**:
>   Dynamic props rely on state, allowing the content or properties to be dynamic, such changing the styling of an element  in response to interactions within an application. Mostly expressed in the form of signals `value={value()}`.
> 
> - **Data transfer**:
>   Props are also used to fill components with data that comes from resources, like [`createResource`](/reference/basic-reactivity/create-resource) calls.
>   This results in components that react in real-time to data changes.

Reviewer: ```suggestion
- **Static props**:
  In Solid's JSX, static props are integrated directly into the HTML by cloning the template and using them as attributes.

 - **Dynamic props**:
 Dynamic props rely on state, allowing the content or properties to be dynamic.
 An example is changing the style of an element in response to interactions within an application. 
 This can be expressed in the form of signals (`value={value()}`).

- **Data transfer**:
  Props are also used to fill components with data that comes from resources, like [`createResource`](/reference/basic-reactivity/create-resource) calls.
  This results in components that react in real-time to data changes.
```
